src/compiler/filesystem.h

```diff
@@ -45,11 +45,25 @@ class Filesystem {
   virtual List<const char*> package_cache_paths() = 0;
 
   virtual bool is_absolute(const char* path) = 0;
+  // The path the non-absolute path is relative to.
+  // On Posix systems this is equal to `cwd`.
+  // On Windows, it can be `cwd`, or a drive (like "c:"), if the path starts with '\' or '/'.
+  virtual const char* relative_anchor(const char* path) {
+    return cwd();
```

Comment:
As discussed, having these defined in the super class is a source of confusion. Maybe just drop them here?